app/routes/confirm.js

```diff
@@ -32,8 +28,19 @@ module.exports = [
   {
     method: 'POST',
     path: '/confirm',
+    options: {
+      validate: {
+        payload: Joi.object({
+          iConfirm: Joi.boolean().invalid(false)
```

Comment:
I don't think this is right. Currently if you check the box you get the error message, but a non-checked box takes you to the confirmation page.

I think this should work: `Joi.string().valid('I confirm').required()`